src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
```

Comment:
Fields should have access modifiers to define their visibility more clearly. In this case, you likely want 'color' to be private.